aws-codeguruprofiler-profilinggroup/src/test/java/software/amazon/codeguruprofiler/profilinggroup/CreateHandlerTest.java

```diff
@@ -138,6 +154,7 @@ public void itSucceedsWithNullPrincipals() {
             @AfterEach
             public void itOnlyCallsCreatePG() {
                 verify(proxy, times(1)).injectCredentialsAndInvokeV2(eq(createPgRequest), any());
+                verify(proxy, times(1)).injectCredentialsAndInvokeV2(eq(addNotificationChannelsRequest), any());
                 verifyNoMoreInteractions(proxy);
             }
         }
```

Comment:
Rather than editing the current set of tests, please add new tests that validate the change that we're making. Notice how, for instance, this addition made the test description inaccurate in two ways:

1. The behavior is no longer "it only calls create profiling group"
2. The expected behavior of setting up the notifications is not documented anywhere

(Same applies to other tests -- in most cases we should be adding, not modifying)